sdks/rust/proto/allocation/allocation.proto

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 Google LLC All Rights Reserved.
```

Comment:
Duplicating the files within our repo seems like it's likely to cause the copies here in the rust sdk to become out of date, as folks will forget to update them. 

Since our builds run inside of a docker container, would it be possible to copy the proto files from their current location next to cargo.toml as part of the build process so that we dynamically put them into the right location?

Reply:
🤔 could gen.sh copy the files here on each run?

Or failing that, could we symlink it?

Reply:
Bumping this comment. What do we think of using gen.sh to re-copy the protos into the directory, as it'll get run whenever the main protos get updated, and we need to update the other SDK clients?

Reply:
Yah I can try that.